scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -79,26 +80,24 @@ workflow GvsBulkIngestGenomes {
             user_defined_sample_id_column_name = GetColumnNames.sample_name_column,  ## NOTE: if no sample_id_column_name has been specified, this is now the <entity>_id column
             vcf_files_column_name = GetColumnNames.vcf_files_column_name,
             vcf_index_files_column_name = GetColumnNames.vcf_index_files_column_name,
-            sample_set_name = sample_set_name
+            sample_set_name = sample_set_name,
     }
 
     call AssignIds.GvsAssignIds as AssignIds {
         input:
             dataset_name = dataset_name,
             project_id = project_id,
-            external_sample_names = read_lines(PrepareBulkImport.sampleFOFN),
-            samples_are_controls = false
+            bulk_import_tsv = PrepareBulkImport.output_tsv,
```

Comment:
I imagined a step here between PrepareBulkImport and GvsAssignIds where we can keep our original interface so make development easier.  See my comment above in AssignIds